.travis.yml

```diff
@@ -1,12 +1,15 @@
 language: java
 sudo: required
-jdk:
-  - oraclejdk8
-  - openjdk8
-env:
-  - NO_GCE_CHECK=true
+matrix:
+  include:
+    - jdk: oraclejdk8
+      env: JAVA_RELEASE=8 JAVA_VERSION=1.8 SCALA_VERSION=2.11 NO_GCE_CHECK=true
+    - jdk: openjdk8
+      env: JAVA_RELEASE=8 JAVA_VERSION=1.8 SCALA_VERSION=2.11 NO_GCE_CHECK=true
+    - jdk: openjdk11
+      env: JAVA_RELEASE=11 JAVA_VERSION=11 SCALA_VERSION=2.12 NO_GCE_CHECK=true
```

Comment:
you can add a an `env: global:` block to always hav NO_GCE_CHECK=true without re-specifying it

```
env:
  global:
    NO_GCE_CHECK=true
```

You can probably combine that onto a single line but I'm never that clear about yaml syntax...

Reply:
Thanks for the suggestion - I'll try that.